src/main/java/picard/sam/util/PhysicalLocation.java

```diff
@@ -1,11 +1,13 @@
 package picard.sam.util;
 
+import java.io.Serializable;
+
 /**
  * Small interface that provides access to the physical location information about a cluster.
  * All values should be defaulted to -1 if unavailable.  ReadGroup and Tile should only allow
  * non-zero positive integers, x and y coordinates may be negative.
  */
-public interface PhysicalLocation {
+public interface PhysicalLocation extends Serializable {
```

Comment:
why does this ned to be serializable?

Reply:
Travis was failing with `java.io.NotSerializableException: picard.sam.util.PhysicalLocationInt` (https://travis-ci.com/github/broadinstitute/picard/jobs/434802169), making PhysicalLocation serializable solved the issue.

Reply:
so a comment, "needs to be Serializable since it is a field in ReadNameParser which is Serializable." would help here.